### PR TITLE
Move switch RIB timeseries definition to TOML

### PR DESCRIPTION
**Pull request by bnaecker**

### CODE REVIEW

File: oximeter/oximeter/schema/switch-rib.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+description = "Maghemite router routing information base"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "hostname", "link_id", "rack_id" ] }
+]
+
+[[metrics]]
+name = "active_routes"
+description = "The number of currently active routes"
+units = "count
```
Comment (rcgoodfellow) [inline oximeter/oximeter/schema/switch-rib.toml:14]: seem to be missing a closing quote here

File: oximeter/oximeter/schema/switch-rib.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+format_version = 1
+
+[target]
+name = "switch_rib"
+description = "Maghemite router routing information base"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "hostname", "link_id", "rack_id" ] }
```
Comment (rcgoodfellow) [inline oximeter/oximeter/schema/switch-rib.toml:8]: is `link_id` supposed to be `sled_id` here?

File: oximeter/oximeter/schema/switch-rib.toml (hunk 1 above)
Comment (bnaecker) [reply oximeter/oximeter/schema/switch-rib.toml:14]: Thanks, fixed in 2823506b00d48c661911faca9781c50e5ed35377

File: oximeter/oximeter/schema/switch-rib.toml (hunk 2 above)
Comment (bnaecker) [reply oximeter/oximeter/schema/switch-rib.toml]: Yeah, my bad, thank you for catching. Fixed in 2823506b00d48c661911faca9781c50e5ed35377

**Review by rcgoodfellow — APPROVED** [review]

**Pull request merged**